src/components/Technologies/PackCardIcon.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from "react";
+import styles from "./PackCardIcon.module.scss";
+import IconMapper from "@site/src/components/IconMapper/IconMapper";
+import { Image } from "antd";
+
+interface PackCardIconProps {
+  title?: string;
+  logoUrl?: string;
+  type?: any;
+}
+
+export default function PackCardIcon({ title, logoUrl, type }: PackCardIconProps) {
+  const [ isError, setIsError ] = useState(false);
+  const handleImageError = (e: any) => {
+    e.target.style.display = "none";
+    setIsError(true);
```

Comment:
it the image did not load, then you should use a placeholder for it.
On the product, we fallback to the pack type logo.